### DISCUSSION TITLE
How to change site title? can't edit settings.py

### DISCUSSION THREAD

**Discussion by cankirca** [WeblateOrg/weblate#11811] [Q&A] [answered]
Hello,
 
 Installed with docker-compose without issue, but can't find how to change site title and some other settings like admin contacts. I understand that it needs to be changed via settings.py. I whent to inside of docker image and there is a file called settings_docker.py, but there is no vi or nano installed on my machine. Is it correct file? Is there a different method?
 Thank you.

**Comment by nijel** [comment, marked as answer]
Please read https://docs.weblate.org/en/latest/admin/install/docker.html

**Comment by cankirca** [comment]
Worked, thank you.
